### DISCUSSION TITLE
Add passive tracer advection-diffusion by a QG flow to `traceradvectiondiffusion.jl`

### DISCUSSION THREAD

**Discussion by jbisits** [FourierFlows/PassiveTracerFlows.jl#49] [Ideas]
I have been using a modified (by myself and Navid) version of the `traceradvectiondiffusion.jl` module to look at passive tracer advection-diffusion by a QG flow. Here is the repo for it:

https://github.com/jbisits/QG_tracer_advection

It uses the `MultiLayerQG.jl` module from `GeophysicalFlows.jl` to setup a multilayer QG flow and advect-diffuse the tracer using this flow.

If you think this would be worthwhile to add to this module (along with an example showing how it works) let me know and I can begin the process. 

Joey

**Comment by navidcy** [comment]
It would be nice to enhance the module to allow this possibility. Let's think about what's the best way to do that.

cc @glwagner
